src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -24,6 +24,14 @@ public static async Task InstallPackage()
             await InstallPipWheel();
             await InstallPythonAzureFunctionPackage();
             await PipFreeze();
+            try
+            {
+                await EnsureVirtualEnvrionmentIgnored();
+            }
+            catch (Exception)
+            {
+                // Safe execution, we aren't harmed by failures here
```

Comment:
can you move the try/catch to the `EnsureVirtualEnvrionmentIgnored()` then? it will make this one a bit more readable.